report-ng/app/src/components/classes/classes.html

```diff
@@ -76,6 +79,20 @@
                     <label>Show configuration methods</label>
                 </mdc-form-field>
             </mdc-layout-grid-cell>
+
+            <!-- Chip List -->
+            <mdc-layout-grid-cell span="12">
+                <mdc-chip-set>
+                    <mdc-chip repeat.for="chip of _chipList"
+                              class="chip-${chip.chipType|chipClass} mr1"
```

Comment:
This logic is to confusing. Consider introducing `IFilterChip.class` property and use in like:
```html
<md-chip class.bind="chip.class` class="mr1">
```